src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -402,36 +362,19 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             var defaults = Default(network:network);
 
             builder.AppendLine("####Node Settings####");
-            builder.AppendLine($"#An optional prefix for the node's user agent shared with peers. Truncated if over { MaximumAgentPrefixLength } characters.");
-            builder.AppendLine($"#agentprefix=<string>");
-            builder.AppendLine($"#Accept non-standard transactions. Default {(defaults.RequireStandard?1:0)}.");
-            builder.AppendLine($"#acceptnonstdtxn={(defaults.RequireStandard?1:0)}");
-            builder.AppendLine($"#Max tip age. Default {network.MaxTipAge}.");
-            builder.AppendLine($"#maxtipage={network.MaxTipAge}");
-            builder.AppendLine($"#Sync with peers. Default 1.");
-            builder.AppendLine($"#synctime=1");
+            builder.AppendLine($"#Test network. Defaults to 0.");
+            builder.AppendLine($"testnet={((network.IsTest() && !network.IsRegTest())?1:0)}");
```

Comment:
Maybe instead of adding !IsRegTest it will be better to change IsTest() to check if name contains `testnet` instead of `test`?

Reply:
That would be assuming that the current implementation of IsTest is incorrect. It may not be.. The purpose may be to check that it is not the main network.

Reply:
Ok, reasonable.